packages/e2e-test-utils-playwright/src/request-utils/pages.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Internal dependencies
+ */
+import type { RequestUtils } from './index';
+
+export type Page = {
+	id: number;
+	status: 'publish' | 'future' | 'draft' | 'pending' | 'private';
+};
+
+/**
+ * Delete all pages using REST API.
+ *
+ * @param {RequestUtils} this
+ */
+export async function deleteAllPages( this: RequestUtils ) {
+	// List all pages.
+	// https://developer.wordpress.org/rest-api/reference/pages/#list-pages
+	const pages = await this.rest< Page[] >( {
+		path: '/wp/v2/pages',
+		params: {
+			per_page: 100,
+			// All possible statuses.
+			status: 'publish,future,draft,pending,private,trash',
```

Comment:
I've noticed an inconsistency between the requested post statuses and the Page type above (missing `trash` status). Maybe we could guard against that and define the statuses as a const context:

```ts
const statuses = [ 'publish', 'future', 'draft', 'pending', 'private', 'trash' ] as const;
```

...then turn it into a status type for Page:

```ts
export type Page = {
	id: number;
	status: typeof statuses[ number ];
}
```

...and finally, join its values to get the `status` rest request param:

```ts
{
	per_page: 100,
	status: statuses.join( ',' ),
}
```

Reply:
Yeah, good idea!